src/Components/Timeline.js

```diff
@@ -0,0 +1,23 @@
+import { onNavigate } from "../index.js";
+
+export const timeline = () => {
+  const content = document.createElement("section");
+
+  const solovinoWorking = document.createElement("img");
+  solovinoWorking.setAttribute("class", "solovino_working_img");
+  solovinoWorking.setAttribute("src", "/Resourses/404-Solovino.png");
+
+  const textTimeline = document.createElement("h1");
+  textTimeline.setAttribute("class", "solovino_working");
+  textTimeline.textContent = "ERROR 404";
+
+  const returnIndex = document.createElement("button");
+  returnIndex.textContent = "Regresa al inicio";
+  returnIndex.setAttribute("id", "return_index");
+  returnIndex.addEventListener("click", () => onNavigate("/"))
+  returnIndex.setAttribute("class", "button");
+  returnIndex.setAttribute("type", "button");
```

Comment:
Esta raro el nombre nooo ??
```suggestion
  const homeButton = document.createElement("button");
  homeButton.textContent = "Regresa al inicio";
  homeButton.setAttribute("id", "return_index");
  homeButton.addEventListener("click", () => onNavigate("/"))
  homeButton.setAttribute("class", "button");
  homeButton.setAttribute("type", "button");
```